bav-ipv-stub/src/handlers/callback.ts

```diff
@@ -0,0 +1,17 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+
+export const handler = async (
```

Comment:
Happy to leave this as is now but let's raise tech debt to make this response not so confusing 🙏

Reply:
Yeah even i was not sure if i should update it as I wanted to understand where was this being used. Created a tech debt ticket to look into this later on: https://govukverify.atlassian.net/browse/KIWI-1345